ensime_shared/ensime.py

```diff
@@ -187,40 +203,89 @@ def ready_to_connect():
             return True
 
         # True if ensime is up and connection is ok, otherwise False
-        return lazy_initialize_ensime() and ready_to_connect()
+        return self.running and lazy_initialize_ensime() and ready_to_connect()
 
     def tell_module_missing(self, name):
+        """Warn users that a module is not available in their machines."""
         msg = feedback["module_missing"]
         self.raw_message(msg.format(name, name))
 
+    def get_runtime_path(self):
+        """Return folder in which the `ensime-vim` plugin is locate."""
+        # Embed path, is there a smarter way to do it?
+        # Note that the path has to be exactly the same
+        # than what 'set runtimepath?' returns
+        return "~/.nvim/plugged/ensime-vim/"
```

Comment:
Ideally this should pull from a variable we can set from our `.vimrc`. For instance, I run out of `~/.config/nvim/plugged/ensime-vim` rather than `~/.nvim/`